bindings/python/tests/compiler_core_test.py

```diff
@@ -53,6 +53,15 @@ def testCompileStrLLVMAOT(self):
     logging.info("Flatbuffer size = %d", len(binary))
     self.assertTrue(binary)
 
+  # Verifies that multiple target_backends are accepted. Which two are not
+  # load bearing.
+  # See: https://github.com/google/iree/issues/4436
+  def testCompileMultipleBackends(self):
+    binary = compiler.compile_str(
```

Comment:
let's not do this - it makes it impossible to disable the backends

Reply:
As discussed on chat, will keep for now, but need to untangle with the new compiler driver.